MixItUp.WPF/Controls/Chat/ChatMessageControl.xaml.cs

```diff
@@ -44,6 +44,12 @@ private void ChatMessageControl_Loaded(object sender, RoutedEventArgs e)
                 this.MessageWrapPanel.Children.Add(this.messageHeader);
             }
 
+            if (this.Message.IsWhisper)
+            {
+                this.Background = (Brush)FindResource("PrimaryHueLightBrush");
```

Comment:
How does this look on Light vs Dark background? Different color schemes? Different full themes? Just want to make sure the text still looks readable in "most" scenarios.

Reply:
Spoke offline, all good here